src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -954,7 +951,7 @@ private void VisitPattern(BoundExpression expression, TypeSymbolWithAnnotations
             }
         }
 
-        protected override BoundNode VisitReturnStatementNoAdjust(BoundReturnStatement node)
+        protected override BoundNode? VisitReturnStatementNoAdjust(BoundReturnStatement node)
```

Comment:
More general question for these: once nullable expands further in the code-base, these overrides are going to need to match in return nullability, right?

Reply:
Specifically, even for the places that we currently have returning `BoundNode` in this file, they'll have to match the parent.

---
In reply to: [241927757](https://github.com/dotnet/roslyn/pull/31719#discussion_r241927757) [](ancestors = 241927757)